changelogs/CHANGELOG-v2.6.rst

```diff
@@ -41,6 +41,7 @@ Bugfixes
 - win_updates - Added the ability to run on a scheduled task for older hosts so async starts working again - https://github.com/ansible/ansible/issues/38364
 - winrm - Add better error handling when the kinit process fails
 - xenserver_facts - ensure module works with newer versions of XenServer (https://github.com/ansible/ansible/pull/35821)
+- vyos - Fixing typo to create easier json failure conditionals using proper spelling. (https://github.com/ansible/ansible/pull/41284)
```

Comment:
This file is generated and should not be edited. If a changelog entry is needed it must be made in a new file in `changelogs/fragments/`. However, this fix does not appear to warrant a changelog entry, so it's OK to just remove it.